dashboard/app/controllers/lti_v1_controller.rb

```diff
@@ -83,34 +94,28 @@ def get_decoded_jwt(integration, id_token)
 
   private
 
+  @namespace = "lti_v1_controller"
```

Comment:
this should probably be a frozen constant, rather than a mutable instance variable